snapshot-labs/sx-monorepo
# feat(ux): add chars count to input with maxLength

### Summary

Display character counter if there is a maxLength limit on a string input.

Closes: #168

![Screenshot 2024-03-21 at 20 05 12](https://github.com/snapshot-labs/sx-monorepo/assets/495709/e3819d71-e0e8-46cd-ad82-757d9f65219b)


### How to test

1. Go to http://localhost:8080/#/create
2. Focus on the `Voting Power Symbol` field
3. It should show `0/12` on the input's top right corner
4. Fill something
5. The chars count should increment, and reflect the input's string length
6. Focus on another input
7. The chars counter should disappear.

NOTE: The chars count will stay displayed if the field is errored
